local-source-proxy/prereqs.hbs.md

```diff
@@ -93,18 +81,16 @@ For example:
 - **Optional**:
```

Comment:
*suggestion*: RE: The Optional part at the end in this page: 
We should have that as a separate heading and instead of saying its optional, we should say For Production use, we highly recommend that you have different secret for Pull as the pull secret will be exported to all developer namespaces by Namespace provisioner and it should not have elevated permissions

Reply:
Added